AsyncDisplayKit/ASCellNode.h

```diff
@@ -167,6 +161,27 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
  */
 - (void)cellNodeVisibilityEvent:(ASCellNodeVisibilityEvent)event inScrollView:(nullable UIScrollView *)scrollView withCellFrame:(CGRect)cellFrame;
 
+#pragma mark - UITableViewCell specific passthrough properties
+
+/* @abstract The selection style when a tap on a cell occurs
+ * @default UITableViewCellSelectionStyleDefault
+ * ASTableView uses these properties when configuring UITableViewCells that host ASCellNodes.
+ */
+@property (nonatomic) UITableViewCellSelectionStyle selectionStyle;
+
+
```

Comment:
@SaifAlDilaimi In the future, I think there's only supposed to be one return before the method comments.

I wonder if we need to set the default values for SelectionStyle or any of the other properties in ASCellNode's init?  I'm not sure what the "0" default corresponds to.  I'll go ahead and merge this for now though.

Thanks for implementing!